exonum-java-binding/core/rust/src/utils/services.rs

```diff
@@ -14,36 +14,46 @@
 
 use std::{collections::HashSet, fs::File, io::Read, path::Path};
 
+use std::collections::HashMap;
 use toml;
 
-pub const PATH_TO_SERVICES_TO_ENABLE: &str = "ejb_app_services.toml";
+pub const PATH_TO_SERVICES_DEFINITION: &str = "ejb_app_services.toml";
 pub const CONFIGURATION_SERVICE: &str = "configuration";
 pub const BTC_ANCHORING_SERVICE: &str = "btc-anchoring";
 pub const TIME_SERVICE: &str = "time";
-pub const EJB_SERVICE: &str = "ejb-service";
 
 #[derive(Serialize, Deserialize)]
-struct ServicesToEnable {
-    services: HashSet<String>,
+pub struct EjbAppServices {
+    pub enabled_services: Option<HashSet<String>>,
+    pub user_services: HashMap<String, String>,
 }
 
 type Result<T> = std::result::Result<T, Box<std::error::Error>>;
 
-/// Loads service names from a specific TOML configuration file
-pub fn load_enabled_services<P: AsRef<Path>>(path: P) -> Result<HashSet<String>> {
+/// Loads services definition from a specific TOML configuration file.
+pub fn load_services_definition<P: AsRef<Path>>(path: P) -> Result<EjbAppServices> {
     let mut file = File::open(path)?;
     let mut toml = String::new();
     file.read_to_string(&mut toml)?;
-    let ServicesToEnable { services } =
-        toml::from_str(&toml).expect("Invalid list of services to enable");
+    parse_services(toml)
+}
+
+// Produces the EjbAppServices structure from string representation in TOML format.
+fn parse_services(data: String) -> Result<EjbAppServices> {
+    let services: EjbAppServices =
+        toml::from_str(&data).expect("Invalid format of the file with EJB services definition");
     Ok(services)
 }
 
 /// Determines whether particular service name is defined in the specific TOML configuration file.
 pub fn is_service_enabled_in_config_file<P: AsRef<Path>>(service_name: &str, path: P) -> bool {
```

Comment:
The documentation of this method must be updated to reflect which services it supports (seemingly not user services).

Reply:
Done.